docs/datatypes/text/index.md

```diff
@@ -190,4 +202,163 @@ print(q.matches[:, ('text', 'scores__jaccard')])
 {'value': 0.6666666666666666, 'ref_id': 'f47f7448709811ec960a1e008a366d49'}, 
 {'value': 0.6666666666666666, 'ref_id': 'f47f7448709811ec960a1e008a366d49'}, 
 {'value': 0.7142857142857143, 'ref_id': 'f47f7448709811ec960a1e008a366d49'}]]
-```
\ No newline at end of file
+```
+
+
+## Searching at chunk level with subindex
```

Comment:
We should somewhere connect this to `multimodal` access.

Reply:
Yes, I think is is still from the very beginning. I will look at this again